mantis-common/src/main/java/io/mantisrx/common/metrics/measurement/Measurements.java

```diff
@@ -38,11 +40,13 @@ public Measurements(
             @JsonProperty("timestamp") long timestamp,
             @JsonProperty("counters") Collection<CounterMeasurement> counters,
             @JsonProperty("gauges") Collection<GaugeMeasurement> gauges,
+            @JsonProperty("micrometers") Collection<MicrometerMeasurement> micrometers,
```

Comment:
Why do we need this when we are already emitting the existing data-structures? Unless the downstream understands this DS, emitting it is unnecessary.

Reply:
The `CounterMeasurement` and `GaugeMeasurement` are for existing mantis internal metrics. All micrometer metrics will be part of the `MicrometerMeasurement` for now.